veros/setup/acc_basic/acc_basic.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+from veros import VerosSetup, veros_method
+from veros.tools import cli
+from veros.variables import allocate
+from veros.distributed import global_min, global_max
+
+
+class ACCBasicSetup(VerosSetup):
+    """A model using spherical coordinates with a partially closed domain representing the Atlantic and ACC.
+
+    Wind forcing over the channel part and buoyancy relaxation drive a large-scale meridional overturning circulation.
+
+    This setup demonstrates:
+     - setting up an idealized geometry
+     - updating surface forcings
+     - constant horizontal and vertical mixing (switched off IDEMIX and GM_EKE)
+     - basic usage of diagnostics 
+    
+    :doc:`Adapted from ACC channel model </reference/setups/acc>`. 
+    """
+    @veros_method
+    def set_parameter(self, vs):
+        vs.identifier = 'acc_basic'
+
+        vs.nx, vs.ny, vs.nz = 30, 42, 15
+        vs.dt_mom = 4800
+        vs.dt_tracer = 86400 / 2.
+        vs.runlen = 86400 * 365 * 20
+
+        vs.coord_degree = True
+        vs.enable_cyclic_x = True
+
+        vs.congr_epsilon = 1e-12
+        vs.congr_max_iterations = 5000
+
+        vs.enable_neutral_diffusion = True
+        vs.K_iso_0 = 1000.0
+        vs.K_iso_steep = 500.0
+        vs.iso_dslope = 0.005
+        vs.iso_slopec = 0.01
+        vs.enable_skew_diffusion = True
+
+        vs.enable_hor_friction = True
+        vs.A_h = (2 * vs.degtom)**3 * 2e-11
+        vs.enable_hor_friction_cos_scaling = True
+        vs.hor_friction_cosPower = 1
+
+        vs.enable_bottom_friction = True
+        vs.r_bot = 1e-5
+
+        vs.enable_implicit_vert_friction = True
+
+        vs.enable_tke = True
+        vs.c_k = 0.1
+        vs.c_eps = 0.7
+        vs.alpha_tke = 30.0
+        vs.mxl_min = 1e-8
+        vs.tke_mxl_choice = 2
+        vs.kappaM_min = 2e-4
+        vs.kappaH_min = 2e-5
+        vs.enable_Prandtl_tke = False
+        vs.enable_kappaH_profile = True
+        # vs.enable_tke_superbee_advection = True
+
+        vs.K_gm_0 = 1000.0
+        vs.enable_eke = False
+        vs.eke_k_max = 1e4
+        vs.eke_c_k = 0.4
+        vs.eke_c_eps = 0.5
+        vs.eke_cross = 2.
+        vs.eke_crhin = 1.0
+        vs.eke_lmin = 100.0
+        vs.enable_eke_superbee_advection = False
+        vs.enable_eke_isopycnal_diffusion = False
+
+        vs.enable_idemix = False
+        vs.enable_idemix_hor_diffusion = False
+        vs.enable_eke_diss_surfbot = False
+        vs.eke_diss_surfbot_frac = 0.2
+        vs.enable_idemix_superbee_advection = False
```

Comment:
Let's just remove all the stuff we're not using, this is supposed to be basic after all :)